app/models/solidus_content/entry_type.rb

```diff
@@ -1,53 +1,63 @@
 # frozen_string_literal: true
 
-class SolidusContent::EntryType < ActiveRecord::Base
-  has_many :entries, dependent: :destroy
+module SolidusContent
```

Comment:
Not blocking, but would be better to keep the namespace change in a separate commit, so that the actual change is more clear